teamengine-core/src/main/java/com/occamlab/te/CtlEarlReporter.java

```diff
@@ -12,6 +12,26 @@
 
 package com.occamlab.te;
 
+/*-
+ * #%L
+ * TEAM Engine - Core Module
+ * %%
+ * Copyright (C) 2006 - 2024 Open Geospatial Consortium
```

Comment:
The start date may be on a file by file basis, does the plugin check gor history?

Reply:
Thanks for the feedback. I agree, this is not correct 100% as some files might be newer than 2006. We will check this.